### PR TITLE
[feat] Add descriptive document titles for pages

### PR DESCRIPTION
**Pull request by KaroMourad**
Add: 
- document titles special for every page

Fix:
-  Badge component key warning

### CODE REVIEW

**Comment by KaroMourad** [comment]
Resolves #1754

**Review by rubenaprikyan — APPROVED** [review]
lgtm!

**Review by arsengit — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -15,6 +16,7 @@
 
 - Resolve issue with rendering run params in the overview tab of SRP (arsengit)
 - Fix issue with search query state update (arsengit)
+>>>>>>> main
```
Comment (rubenaprikyan) [inline CHANGELOG.md:19]: Typo after merging

Comment (KaroMourad) [reply CHANGELOG.md]: resolved

**Review by rubenaprikyan — APPROVED** [review]

**Pull request merged**